public/i18n/en/translation.json

```diff
@@ -1028,6 +1041,65 @@
     "vpn": {
       "title": "VPN"
     },
+    "openvpn_rw": {
+      "title": "OpenVPN Road Warrior",
+      "roadwarrior_server": "RoadWarrior server",
+      "rw_server": "RW server",
+      "roadwarrior_server_description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+      "no_openvpn_rw_server_found": "No OpenVPN RoadWarrior server found",
+      "create_server": "Create server",
+      "edit_server": "Edit server",
+      "server_name": "Server name",
+      "user_database": "User database",
+      "create_accounts_for_all_users": "Create account for all users",
+      "create_accounts_for_all_users_tooltip": "All accounts created will have a certificate valid for 3650 days.",
+      "authentication_mode": "Authentication mode",
+      "mode": "Mode",
+      "routed": "Routed",
+      "bridged": "Bridged",
+      "vpn_network": "VPN network",
+      "vpn_network_tooltip": "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+      "public_ip_hostname_unit": "Public IP / hostname of this unit",
+      "add_public_ip_host": "Add public IP/host",
+      "protocol": "Protocol",
+      "udp": "UDP",
+      "tcp": "TCP",
+      "auto": "Auto",
+      "server_client_negotiation": "Server/client negotiation",
+      "port": "Port",
+      "route_all_client_traffic_through_vpn": "Route all client traffic through VPN",
+      "push_custom_network_routes": "Push custom networks routes",
+      "bridge": "Bridge",
+      "range_ip_start": "Range IP start",
+      "range_ip_end": "Range IP end",
+      "add_network": "Add network",
+      "allow_client_to_client_network_traffic": "Allow client-to-client network traffic",
+      "compression": "Compression",
+      "digest": "Digest",
+      "cipher": "Cipher",
+      "enforce_minimum_tls_version": "Enforce a minimum TLS version",
+      "custom_options": "Custom options",
+      "custom_options_tooltip": "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+      "create": "Create",
+      "add_option": "Add option",
+      "delete_server_rw": "Delete server RW",
+      "delete_server_rw_message": "This action will also delete all account for this server.",
+      "database": "Database",
+      "status": "Status",
+      "connected_clients": "Connected clients",
+      "roadwarrior_accounts": "RoadWarrior accounts",
+      "roadwarrior_accounts_description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+      "username_password": "Username and password",
+      "certificate": "Certificate",
+      "username_password_certificate": "Username, password and certificate",
+      "username_otp_certificate": "Username and OTP certificate",
+      "local_database": "Local database",
+      "custom_option_combobox_placeholder": "Search key",
+      "strong": "Strong",
+      "weak": "Weak",
+      "server_configured": "Server configured",
+      "roadwarrior_server_configured_successfully": "RoadWarrior server configured successfully"
```

Comment:
```suggestion
      "roadwarrior_server_configured_successfully": "Road Warrior server configured successfully"
```

Reply:
This item was directly removed after talking with Andrea, we'll be using a single title for the toast instead